modules/custom/social_language/social_language.install

```diff
@@ -114,3 +117,11 @@ YAML;
     'translate topic_types taxonomy_term',
   ]);
 }
+
+/**
+ * Allow "Content Manager" role to manage translations.
+ */
+function social_language_update_10301() {
+  // The function grand permissions.
+  _social_language_set_permissions();
```

Comment:
Update hooks should [be explicit](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_update_N/8.2.x) in changing from one state to another. The `_social_language_set_permissions` function is used to set the module's default permissions and may change over time. This makes the update hook unstable.

Reply:
Added changes